style/reading.less

```diff
@@ -10,59 +10,111 @@
 @import 'reading/video';
 @import 'reading/links-to-physics';
 @import 'reading/work-in-physics';
-// @import 'reading/tips-for-success';
+@import 'reading/tips-for-success';
 // @import 'reading/grasp-check';
 
 .task-step .panel-body {
 
-  .reading-step .section-opener .learning-objectives {
-    width: 100%;
-    min-height: 100px;
-    display: inline-flex;
-    color: #ffffff;
-    padding: 10px 40px;
-    background: @physics-background-secondary;
+  .reading-step {
+
+    div[data-type="document-title"] {display: none;}
+
+    .splash {
+      width: 100%;
+      margin: -20px 0 0 0;
+
+      img {
+        border: 0;
+        margin: 0;
+      }
+
+      figcaption {margin: 0 40px;}
+
+      + p {
+        clear: both;
+        padding-top: 40px;
+      }
 
-    > div {
-      display: none
     }
 
-    p {
-      color: rgba(255,255,255,.75);
-      #fonts > .sans;
-      text-transform: uppercase;
-      text-align: right;
-      margin: auto 15px;
+    figure {
```

Comment:
For a future PR: would it make sense for this styling to be scoped inside `.arbitrary-html`? That way it can apply to other book content? like markup inside a video step, or for the reference book.

Or, you could do `.reading-step .arbitrary-html { .book-styles(); }`